merlin/dag/utils.py

```diff
@@ -67,3 +71,61 @@ def group_values_offsets(flat_cols: dict) -> dict:
             grouped_cols[key] = value
 
     return grouped_cols
+
+
+class ModelRegistry(ABC):
```

Comment:
I don't think this concept belongs in Core—it could maybe become the seed of the Merlin Ops/Orchestration library we've talked about for a while now?

Reply:
@nv-alaiacano, do we have enough between the loose ends in Systems and the code/learning from AI Workflows to serve as a seed crystal for a new Merlin library?